docs/framework/social-post-generator.mdx

```diff
@@ -2,218 +2,410 @@
 title: "Social post generator"
 ---
 
-In this tutorial, you’ll use the Writer Framework to build an application that generates social media posts and tags based on user input.
+In this tutorial, you'll create a social post generator using the Writer Framework.
+
+You'll build an AI-powered tool for generating social media posts and tags based on the input you provide! You'll do this in minutes using a drag-and-drop visual editor to build the user interface and Python for the back-end code.
+
+Here's what the finished project will look like:
 
 ![Finished social post generator application](/framework/images/tutorial/social_post/sp_gen_1.png)
 
 
+## Prerequisites
+
+Before starting, ensure you have:
+
+- **A Writer account:** Create a free account at [writer.com](https://writer.com).
+- **Python 3.7+**: Use the installer from [python.org](https://www.python.org/downloads/).
```

Comment:
It says 3.9.2 or higher on the Quickstart, probably should change that here

Reply:
Updated.